brut.apktool/apktool-lib/src/main/java/brut/androlib/ApkDecoder.java

```diff
@@ -53,7 +53,7 @@ public class ApkDecoder {
     private final static String SMALI_DIRNAME = "smali";
     private final static String UNK_DIRNAME = "unknown";
     private final static String[] APK_RESOURCES_FILENAMES = new String[] {
-        "resources.arsc", "AndroidManifest.xml", "res", "r", "R" };
+        "resources.arsc", "res", "r", "R" };
```

Comment:
Any background on removal of `AndroidManifest.xml` from this? In my vague memory we needed to duplicate it here so in a raw build (pure copy of compiled assets) we needed to include the Manifest.

Reply:
This private field used only in the decoder. In the builder has different list - don't touched

Reply:
I think need make class ApkArchive and method analyze():
a.  fill full entry list of the Apk
b. split this list to the category
  1. AndroidManifest.xml
  2. META-INF/ directory - with services
  3. resources.asrc
  4. resource files - liked from resources,asrc
  5. uncompressed files
 
  7. other or unknown files

Reply:
To work the features needed for `--analysis-mode` ?

Reply:
Yes, but we have use this splitted list for copy items. Don't need NO_COMPRESS_PATTERN - this information has in the apk.